app/utils/openlayers.SearchControl.js

```diff
@@ -2,15 +2,841 @@ import openlayers from 'openlayers'
 
 export default class SearchControl extends openlayers.control.Control {
   constructor(className) {
+
     const element = document.createElement('div')
     super({element})
     element.className = `${className || ''} ol-unselectable ol-control`
     element.title = 'Jump to a specific location or coordinate'
+    element.id = 'searchCoordBtn'
```

Comment:
Can we remove this ID?  That way we can avoid a collision if the control is ever reused.